config/config.go

```diff
@@ -77,19 +78,42 @@ func load0(configDir string, tags []string) (*Config, error) {
 		Dir:  configDir,
 		Tags: tags,
 	})
-	for _, inst := range insts {
-		if err := inst.Err; err != nil {
-			return nil, fmt.Errorf("cannot load CUE instances in %q: %w", configDir, err)
-		}
+
+	if len(insts) != 1 {
+		return nil, fmt.Errorf("wrong instance count")
 	}
-	vals, err := ctx.BuildInstances(insts)
-	if err != nil {
-		return nil, fmt.Errorf("cannot build instances: %w", err)
+
+	inst := insts[0]
+	if err := inst.Err; err != nil {
+		return nil, fmt.Errorf("cannot load CUE instance in %q: %w", configDir, err)
 	}
-	if len(vals) != 1 {
-		return nil, fmt.Errorf("wrong value count")
+
+	for _, path := range paths {
+		if fi, err := os.Stat(path); err != nil {
+			if os.IsNotExist(err) {
+				return nil, fmt.Errorf("cannot load CUE path %q: path does not exist", path)
+			}
+			return nil, fmt.Errorf("cannot load CUE path %q: %w", path, err)
+		} else if fi.IsDir() {
+			return nil, fmt.Errorf("CUE path must be a file, found directory %q", path)
+		}
+
+		by, err := os.ReadFile(path)
+		if err != nil {
+			return nil, fmt.Errorf("cannot read CUE path %q: %w", path, err)
```

Comment:
I'm pretty sure the error will already have the path in, so this will stutter.

```suggestion
			return nil, fmt.Errorf("cannot read CUE file: %w", path, err)
```